README.md

```diff
@@ -65,6 +65,16 @@ directory".
 </details>
 </blockquote>
 
+### Custom API Endpoints
+
+By default, Snyk uses the https://snyk.io/api endpoint. 
+It is possible to configure Snyk to use a different endpoint by changing the `SNYK_API` environment variable:
```

Comment:
Let's link Snyk CLI Docs in case this documentation gets stale:
https://docs.snyk.io/snyk-cli/configure-the-snyk-cli#configuration-to-connect-to-the-snyk-api